internal/cli/atlas/backup/restores_start.go

```diff
@@ -156,8 +143,27 @@ func RestoresStartBuilder() *cobra.Command {
 		ValidArgs: []string{automatedRestore, downloadRestore, pointInTimeRestore},
 		Annotations: map[string]string{
 			"args":             "deliveryType",
+			"requiredArgs":     "deliveryType",
 			"deliveryTypeDesc": "Type of restore job to create. Accepted values include: automated, download, pointInTime.",
 		},
+		Example: fmt.Sprintf(`The following example creates an automated restore
+  $ %[1]s backup restore start automated \
```

Comment:
Not sure if this file requires the same changes as above. If so, please make them here too.